pkg/ui/workspaces/cluster-ui/src/util/hooks.ts

```diff
@@ -193,3 +194,28 @@ export const useSwrMutationWithClusterId = <
   const keyWithClusterId = useSwrKeyWithClusterId(key) as SWRKey;
   return useSWRMutation(keyWithClusterId, fetcher, config);
 };
+
+export const usePagination = (
```

Comment:
nit: Should we add a doctsrinng like the other methods?

Reply:
Done.